mlos_bench/mlos_bench/config/services/remote/azure/scripts/setup-rg/README.md

```diff
@@ -0,0 +1,68 @@
+# Resource Group Setup
+
+These scripts are helpers for setting up a new resource group (RG) in Azure as the *control plane* for MLOS.
+The *control plane RG* is a container for the *persistent* resources of MLOS (results/metrics storage, scheduler VM, notebook interface, etc.).
+
+## Quickstart
+
+1. Starting in this current directory, ensure that we are logged in to Azure CLI.
+
+    ```sh
+    az login
+    ```
+
+1. Make a copy of the control plane ARM parameters file.
+
+    ```sh
+    cp rg-template.example.parameters.json rg-template.parameters.json
+    ```
+
+1. (Optional) Make a copy of the results DB parameters file, if planning to provision a results DB.
+
+    ```sh
+    cp results-db/mysql-template.parameters.example.json results-db/mysql-template.parameters.json
+    ```
+
+1. Modify the ARM parameters in the newly created files as needed, especially the `PLACEHOLDER` values.
+
+1. Execute the main script with CLI args as follows:
+
+    ```shell
+    ./setup-rg.ps1 `
```

Comment:
Please provide alternative examples for sh version as well.

Reply:
Added sh example!